docs/sdk/authentication/auth-sig.md

```diff
@@ -101,12 +101,15 @@ const siwe = require('siwe');
 
 async function main() {
   // Initialize LitNodeClient
-  const litNodeClient = new LitJsSdk.LitNodeClientNodeJs();
+  const litNodeClient = new LitJsSdk.LitNodeClientNodeJs({
+		alertWhenUnauthorized: true,
```

Comment:
Do we want this to be true?  This produces a js alert() when unauthorized.  Most users don't want this so it's off by default.  I think it could be omitted?

Reply:
Good point, just used that for my testing.
Should i rather omit the parameter entirely, or set it to false like in the [installation docs](https://developer.litprotocol.com/v3/sdk/installation#sdk-installed-via-nodejs--serverside-usage)?